src/compute_horde_prompt_solver/run.py

```diff
@@ -45,6 +49,17 @@ def parse_arguments():
     parser.add_argument(
         "--seed", type=int, default=42, help="Random seed for reproducibility"
     )
+    parser.add_argument(
+        "--server",
+        action="store_true",
+        help="Spin up a temporary HTTP server to receive the seed",
+    )
```

Comment:
Should seed and server be mutually exclusive? Currently passing both seed and server would make it use the server and ignore the seed.